code/game/gamemodes/changeling/powers/shriek.dm

```diff
@@ -46,4 +46,5 @@
 		L.on = 1
 		L.break_light_tube()
 	empulse(get_turf(user), 2, 4, 1)
+	user.SetConfused(0)
```

Comment:
Might be a good idea to leave a code comment to *why* this is done and what it is trying to avoid.

Reply:
Agreed, fixing.